test/rules/APIGW.test.ts

```diff
@@ -332,4 +324,189 @@ describe('Amazon API Gateway', () => {
       validateStack(stack, ruleId, TestType.COMPLIANCE);
     });
   });
+
+  describe('APIGWStructuredLogging: API Gateway stages use JSON-formatted structured logging', () => {
+    const ruleId = 'APIGWStructuredLogging';
+
+    test('Noncompliance 1: Non-JSON format (CfnStage)', () => {
```

Comment:
Please order these so all the noncompliance tests come before the compliance tests